plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -2742,6 +2742,17 @@ chain::symbol read_only::extract_core_symbol()const {
    return core_symbol;
 }
 
+read_only::get_consensus_parameters_results
+read_only::get_consensus_parameters(const get_consensus_parameters_params& ) const {
+   get_consensus_parameters_results results;
+
+   results.chain_config = db.get_global_properties().configuration;
+   results.kv_database_config = db.get_global_properties().kv_configuration;
```

Comment:
Remove this line. `kv_configuration` should not exist and Mandel and needs to be removed (or repurposed in the future).

Reply:
It's already in mandel ever since https://github.com/eosnetworkfoundation/mandel/pull/8. Should an issue be made for its removal?